### DISCUSSION TITLE
How to request a custom firmware build

### DISCUSSION THREAD

**Discussion by RobbieRw** [mriscoc/Ender3V2S1#665] [General]
Hi man

First of all wow great build! with marlin 2.1.2 is it possible for you to turn on the input shaping please? i want to get my speeds up.

Also can you remove the limit of files that can be held on the micro sd card please. i have over 200 GCODE Files and i can only see quarter of them.

**Comment by RobbieRw** [comment]
** Ender 3 V2

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Special_Configurations

**Comment by RobbieRw** [comment]
Hi

Are you able to compile a fw version for ender 3 v2 that doesnt have the
limit of files that can be viewed on the memory card please?

Also are you able to configure input shaping. I want to be able to get
higher print speeds with my ender 3 v2.

Thanks

**Reply by mriscoc** [reply]
Please send your request to the email address of the message received when you made your donation through Paypal or Patreon, the request will be attended following: https://github.com/mriscoc/Special_Configurations/blob/main/README.md#these-special-configurations-and-releases-are-sponsored-by-donors
